cmake/CMakeLists.txt

```diff
@@ -984,9 +984,12 @@ function(onnxruntime_set_compile_flags target_name)
       foreach(FLAG ${ORT_WARNING_FLAGS})
         target_compile_options(${target_name} PRIVATE "$<$<COMPILE_LANGUAGE:CUDA>:SHELL:--compiler-options ${FLAG}>")
       endforeach()
-      if ((NVCC_HAS_STRICT_ALIASING AND "${target_name}" MATCHES "cuda") OR (HAS_STRICT_ALIASING AND NOT "${target_name}" MATCHES "cuda"))
+      if (${NVCC_HAS_STRICT_ALIASING} AND "${target_name}" MATCHES "cuda")
```

Comment:
There is build error, use original naming?
${NVCC_HAS_STRICT_ALIASING} => NVCC_HAS_STRICT_ALIASING
${HAS_STRICT_ALIASING} => HAS_STRICT_ALIASING

Reply:
Done I was not aware that it makes a big change. I though those "variable evaluation brackets" would actually be safer.